app/controllers/application_controller/buttons.rb

```diff
@@ -252,27 +252,12 @@ def ab_group_delete
 
   private ###########
 
+  EXPLORER_CLASSES = [Vm, MiqTemplate, Service, ServiceTemplate]
   def custom_buttons
     button = CustomButton.find_by_id(params[:button_id])
-    case button.applies_to_class
-    when "Vm"
-      @explorer = true
-      cls = Vm
-    when "MiqTemplate"
-      @explorer = true
-      cls = MiqTemplate
-    when "Host"
-      cls = Host
-    when "EmsCluster"
-      cls = EmsCluster
-    when "ExtManagementSystem"
-      cls = ExtManagementSystem
-    when "Service", "ServiceTemplate"
-      @explorer = true
-      cls = Service
```

Comment:
"ServiceTemplate" was being converted to class `Service`.  This PR changes it to be class `ServiceTemplate`.  Has that change been validated?